transactional_licence_form/forms.py

```diff
@@ -186,80 +187,83 @@ class WorkingPractices2Form(FCLForm):
+        label="25. Do you intend to use computational analysis to do any of the following?",
         help_text="Select all that apply.",
         choices=list_to_choices(choices.COMPUTATIONAL_ANALYSIS_TYPE_CHOICES),
     )
     generative_ai_use = fields.FCLChoiceField(
-        label="25. Will you notify people when they are using generative AI services or content?",
+        label="26. Will you notify people when they are using generative AI services or content?",
         choices=list_to_choices(choices.GENERATIVE_AI_USE_CHOICES),
     )
     explain_limitations = fields.FCLChoiceField(
         label=(
-            "26. Will you explain how the limits of the find case law "
-            "collection impacts your computational analysis to users?"
+            "27. Will you explain how the limits of the Find Case Law "
+            "collection impacts the outcomes of your computational analysis?"
         ),
         choices=list_to_choices(choices.YES_NO_CHOICES),
     )
 
 
 class NinePrinciplesAgreementForm(FCLForm):
-    title = "Statements and Principles"
+    title = "Step 8 - 9 Principles"
 
     accept_principles = fields.FCLChoiceField(
         label=(
-            "27. Licence holders must acknowledge and abide by all the 9 principles. "
-            "Do you accept this licence term?"
+            "28. Licence holders must acknowledge and abide by all the 9 principles."
+            "Do you accept all 9 principles as licence terms?"
         ),
         choices=list_to_choices(choices.YES_NO_CHOICES),
     )
 
 
 class NinePrinciplesStatementForm(FCLForm):
-    title = "Statements and Principles"
+    title = "Step 9 - 9 Principles Statement"
 
     def layout(self):
         return Layout(
             Field.textarea("principles_statement"),
         )
 
     principles_statement = fields.FCLCharField(
-        label="28. Please describe how you will meet the 9 principles",
-        help_text="You should identify any risks and how you will address these risks for each of the 9 principles.",
-        widget=forms.Textarea(),
+        label="29. Please describe how you will meet the 9 principles as terms.",
+        widget=forms.Textarea(attrs={"maxlength": 5000}),
     )
 
 
 class AdditionalCommentsForm(FCLForm):
-    title = "Additional Comments"
+    title = "Step 10 - Additional vomments"
 
     def layout(self):
         return Layout(
             Field.textarea("additional_comments", max_words=250),
         )
 
     additional_comments = fields.FCLCharField(
-        label="29. Are there any additional comments you would like to make in relation to your application?",
+        label="30. Are there any additional comments you would like us to consider as part of your application?",
         help_text="This question is optional.",
-        widget=forms.Textarea(attrs={"data-maxwords": 250}),
+        widget=forms.Textarea(attrs={"data-maxwords": 250, "maxlength": 1500}),
```

Comment:
That's quite a low maxlength for the number of words they might write.

Reply:
This is the one I used 6 letter words on average for!